### ISSUE TITLE
UnicodeDecodeError: 'cp950' codec

### ISSUE THREAD

**Issue by b10632034** [tmilovanov/wisecreator#32]
Hi,
i have encountered some problems when running 'wisecreator.exe' on window 10. Any thought?

```
[.] Checking dependenices
[.] Converting mobi 2 mobi to generate ASIN
[.] Getting ASIN
[.] Getting rawml content of the book
[.] Collecting words
[.] Count of words: 45754
Traceback (most recent call last):
  File "main.py", line 395, in <module>
  File "main.py", line 359, in main
UnicodeDecodeError: 'cp950' codec can't decode byte 0xe2 in position 6708: illegal multibyte sequence
[8108] Failed to execute script main
```

**Comment by tmilovanov** [comment]
Hi, @b10632034!
It happens sometimes with books having encoding different from utf-8. Can you send me the book so that I can debug this? You can send it to the email timcess@gmail.com

**Comment by b10632034** [comment]
Okay

**Comment by tmilovanov** [comment]
Fixed in v1.1
